test/util/server/server.go

```diff
@@ -395,6 +401,46 @@ func serviceAccountSecretsExist(client *kclient.Client, namespace string, sa *ka
 	return foundTokenSecret && foundDockercfgSecret
 }
 
+// WaitForPodCreationServiceAccounts ensures that the service account needed for pod creation exists
+// and that the cache for the admission control that checks for pod tokens has caught up to allow
+// pod creation. A test pod can be passed to test admission creation. If nil, a default pod is used
+// for the test.
+func WaitForPodCreationServiceAccounts(client *kclient.Client, namespace string, serviceAccounts []string, testPod *kapi.Pod) error {
```

Comment:
> Because you may be testing specific conditions for pods that would cause a simple pod to fail.

Add it when we need it?  Seems extremely specific.